contributing/notebook_template/notebook_template.ipynb

```diff
@@ -215,96 +235,74 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "# Get the path to the file\n",
     "file = output['Local Path'][0]\n",
+    "\n",
+    "# Print out some information about our file\n",
     "print('The HDU list of our output file:\\n')\n",
     "print(fits.info(file))\n",
     "\n",
-    "data = fits.getdata(file, 1)['FLUX']\n",
-    "time = fits.getdata(file, 1)['TIME']"
+    "# Load in the flux and time data\n",
+    "time = fits.getdata(file, 1)['time']\n",
+    "flux = fits.getdata(file, 1)['pdcsap_flux']"
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "tags": []
+   },
    "source": [
-    "## Visualization\n",
+    "## Visualization, where relevant\n",
```

Comment:
Also Visualization is under Main Content so:
`### Visualization, where relevant`